src/winetricks

```diff
@@ -6825,13 +6825,11 @@ load_dxsdk_jun2010()
 # $1 - dxvk archive name (required)
 # $2 - dxvk d3d10_enabled / d3d10_disabled (required)
 # $3 - minimum Wine version (optional)
-# $4 - minimum Vulkan API version (optional, requires $3 be set)
```

Comment:
I'm still not clear on why this needs to go. It doesn't show for everyone, only if it fails the check, which to me is a good thing. Is the check broken in some way? Can we fix it instead?

Reply:
@austin987 

Since I can't get a clear answer from Phillip around the Vulkan version requirements... He talks in terms of required extensions. I gather this is mainly an issue around graphics drivers - that may implement extensions, required for DXVK, but do so on top of older versions of Vulkan (that wouldn't normally support these additional extensions)...

Nobody is going to have a good experience with a recent version of DXVK and Wine 3.x (whatever)... Vulkan 1.1.88 is very out-of-date vs. the very recent driver requirements DXVK imposes on Mesa and the Nvidia driver.

I could suggest Wine and Vulkan versions, based closely matching the Vulkan headers that Wine pulls in and that DXVK pulls in. This would be a "recommended target" (in that DXVK may not use all the features of the Vulkan header version being pulls in) - but at least it would be a meaningful guideline. Rather than the current warning that is displayed and has little basis in reality... 

Thoughts?!
Bob

Reply:
Makes sense, thanks.